spec/dummy/spec/integration/user_commands_spec.rb

```diff
@@ -18,4 +18,13 @@
       expect(result.error.messages[:name]).to include("can't be blank")
     end
   end
+
+  describe 'delete' do
+    it 'deletes record' do
+      users.create.call(name: 'Piotr')
+      result = users.try { delete(:by_name, 'Piotr') }
+
+      expect(result.error).to be(nil)
```

Comment:
I'm not sure if this is proper expectation, since it throws error and I don't know what should be the correct output.